docs/pages/getting-started/quick-start.md

```diff
@@ -23,6 +27,45 @@ dojoup
 
 For full `dojoup` reference and debugging see [Dojoup](/toolchain/dojoup.md).
 
+#### Option 2: Using ASDF with asdf-dojo plugin
+
+1. Ensure you have asdf installed. If not, follow the [asdf installation instructions](https://asdf-vm.com/guide/getting-started.html).
+2. Add the asdf-dojo plugin:
+
+```sh
+asdf plugin add dojo https://github.com/dojoengine/asdf-dojo
+```
+
+3. Install the latest version of Dojo or a specific version you need:
+
+```sh
+asdf install dojo latest      # For the latest version
+asdf install dojo 1.2.3       # For a specific version
+```
+4. Set the global or local version:
+
+```sh
+asdf global dojo latest       # Set globally
+asdf local dojo 1.2.3         # Set locally in your project directory
+```
```

Comment:
i think because this is a quick start, we should only show the simplest way to install the latest version.

so imo simply showing this few lines for the asdf step are enough:

```
asdf plugin add dojo https://github.com/dojoengine/asdf-dojo
asdf install dojo latest
asdf global dojo latest
```

Reply:
and we can defer the full asdf commands on a separate more detailed explanation on the installation process.